File: app/src/main/java/com/lazylee/apiguidedemo/coretopics/ui_navigation/canvas/CanvasTwoActivity.java
package com.lazylee.apiguidedemo.coretopics.ui_navigation.canvas;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;

import com.lazylee.apiguidedemo.coretopics.ui_navigation.custom.MyCanvasView;

public class CanvasTwoActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        MyCanvasView myCanvasView;
        myCanvasView = new MyCanvasView(this);
        myCanvasView.setSystemUiVisibility(View.SYSTEM_UI_FLAG_FULLSCREEN);
        setContentView(myCanvasView);
    }
}

File: app/src/main/java/com/lazylee/apiguidedemo/coretopics/ui_navigation/canvas/CanvasTwoActivity.kt
package com.lazylee.apiguidedemo.coretopics.ui_navigation.canvas

import android.os.Bundle
import android.view.View
import androidx.appcompat.app.AppCompatActivity
import com.lazylee.apiguidedemo.coretopics.ui_navigation.custom.MyCanvasView

class CanvasTwoActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        val myCanvasView: MyCanvasView = MyCanvasView(this)
        myCanvasView.systemUiVisibility = View.SYSTEM_UI_FLAG_FULLSCREEN
        setContentView(myCanvasView)
    }
}